src/api/app/assets/javascripts/webui/live_build_log.js

```diff
@@ -95,29 +103,58 @@ $.extend(LiveLog.prototype, {
     this.startButton.addClass('d-none');
   },
 
-  stopAjaxRequest: function() { // jshint ignore:line
+  stopAjaxRequest: function() {
     if (this.ajaxRequest !== 0)
       this.ajaxRequest.abort();
     this.ajaxRequest = 0;
   },
 
-  showAbort: function() { // jshint ignore:line
+  showAbort: function() {
     $(".link_abort_build").removeClass('d-none');
     $(".link_trigger_rebuild").addClass('d-none');
   },
 
-  hideAbort: function() { // jshint ignore:line
+  hideAbort: function() {
     $(".link_abort_build").addClass('d-none');
     $(".link_trigger_rebuild").removeClass('d-none');
   },
 
-  indicatorStatus: function(status) { // jshint ignore:line
+  indicatorStatus: function(status) {
     this.logInfo.children().hide();
     this.logInfo.children('.' + status).show();
   },
 
-  faviconStatus: function(source) { // jshint ignore:line
+  faviconStatus: function(source) {
     if (typeof source !== 'undefined')
       $("link[rel='shortcut icon']").attr("href", source);
   },
+
+  browserNotification: function(status) {
+    if (this.notificationSupport() && Notification.permission === "granted") { // jshint ignore:line
+      var statusDetails = this.buildStatusDetails(status);
+      new Notification('OBS: ' + statusDetails.content, // jshint ignore:line
+        { body: 'The status of your Open Build Service build is "' + statusDetails.indicator + '".', icon: statusDetails.favicon }
+      );
+    }
+  },
+
+  askNotificationPermission: function () {
+    if (this.notificationSupport() && Notification.permission === 'default') { // jshint ignore:line
+      Notification.requestPermission().then(function (permission) { // jshint ignore:line
+        if (permission === 'granted' || permission === 'denied') {
+          liveLog.notificationsButton.html(liveLog.notificationsButton.html().replace('Request Browser Notifications', 'Browser Notifications ' + Notification.permission)); // jshint ignore:line
+          liveLog.notificationsButton.addClass('disabled'); // jshint ignore:line
+        }
+      });
+    }
+  },
+
+  notificationSupport: function () {
```

Comment:
Whenever we start having more browser notifications, this should be extracted to another file. For now, it's good enough.